### PR TITLE
data: bkpecho

### PR DESCRIPTION
**Pull request by bkpecho**
## Check List (Check all the applicable boxes) 

- [x] My code follows the code style of this project.
- [x] My change requires changes to the documentation.
- [x] I have updated the documentation accordingly.
- [x] All new and existing tests passed.
- [x] This PR does not contain plagiarized content.
- [x] The title of my pull request is a short description of the requested changes.

## Note to reviewers

<a href="https://gitpod.io/#https://github.com/EddieHubCommunity/LinkFree/pull/5441"></a>

### CODE REVIEW

**Review by Sindhuinti — APPROVED** [review]
Thanks 🎉

File: data/bkpecho.json [hunk 1]
```diff
@@ -0,0 +1,22 @@
+{
+	"name": "BK Pecho",
+	"type": "personal",
+	"bio": "👨🏽‍💻 Software Engineer at 100Devs | 🧙‍♂️ Web Wizard | 🌱 Open Source Advocate | 🤖 AI Explorer",
+  "avatar": "https://github.com/bkpecho.png",
```
Comment (imbeshat) [inline data/bkpecho.json:5]:
```suggestion
```

Comment (imbeshat) [reply data/bkpecho.json:5]: not required

**Review by schmelto — APPROVED** [review]

**Review by SurajPratap10 — APPROVED** [review]
looks good. Thanks for contributing!!

**Review by Panquesito7 — APPROVED** [review]
Thanks! 🎉

**Review by SaraJaoude — APPROVED** [review]
looks good

**Pull request merged**